cupy/core/core.pyx

```diff
@@ -338,7 +338,7 @@ cdef class ndarray:
 
     # TODO(okuta): Implement byteswap
 
-    cpdef ndarray copy(self, order='C'):
+    cpdef ndarray copy(self, order='K'):
```

Comment:
The default value of `numpy.ndarray.copy`'s `order` option is 'C', while that of `numpy.copy` is 'K'.
https://docs.scipy.org/doc/numpy/reference/generated/numpy.copy.html
https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.copy.html#numpy.ndarray.copy